lib/boom_notifier.ex

```diff
@@ -68,44 +23,17 @@ defmodule BoomNotifier do
       end
 
       # Notifiers validation
-      walkthrough_notifiers(
-        unquote(config),
-        fn notifier, options -> validate_notifiers(notifier, options) end
+      BoomNotifier.Api.walkthrough_notifiers(
+        config,
+        fn notifier, options -> BoomNotifier.Api.validate_notifiers(notifier, options) end
       )
 
-      def notify_error(conn, %{kind: :error, reason: %mod{}} = error) do
-        settings = unquote(config)
-        {ignored_exceptions, _settings} = Keyword.pop(settings, :ignore_exceptions, [])
-
-        unless Enum.member?(ignored_exceptions, mod) do
-          do_notify_error(conn, settings, error)
-        end
+      def walkthrough_notifiers(callback) do
```

Comment:
Looks like this function is not used? :thinking:

Reply:
you're right!